frontend/components/commons/results/ResultsList.vue

```diff
@@ -16,32 +16,35 @@
   -->
 
 <template>
-  <div class="list">
+  <div class="content">  
     <slot name="header" />
     <div class="results-scroll" id="scroll">
       <div :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }" v-if="showLoader">
         <results-loading :size="dataset.viewSettings.pagination.size" />
       </div>
       <DynamicScroller
-        v-else
         page-mode
         class="scroller"
         :items="visibleRecords"
         :min-item-size="150"
         :buffer="200"
         :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }"
+        :emitUpdate="true"
```

Comment:
Do we need this enable this ? Regarding documentation, could penalizes performance:

> emitUpdate (default: false): emit a 'update' event each time the virtual scroller content is updated (can impact performance).